### DISCUSSION TITLE
How to remove the wrapping span?

### DISCUSSION THREAD

**Discussion by 27shutterclicks** [dvtng/react-loading-skeleton#197] [Q&A] [answered]
I see the skeleton's output includes a wrapping span which in my project causes layout issues. If I remove the span, everything works well. 

How can I output the skeleton without the wrapping span?

Thanks.

**Comment by srmagura** [comment, marked as answer]
I recommend using the `containerClassName` prop to style the container `span` so that it does not cause layout issues in your app.

Alternatively, you can import our CSS file and then write 

```tsx
<span className="react-loading-skeleton" />
```

That will give you just the "inner element" with the container `span`. Of course, you are no longer using our React component so you won't be able to use the props that it provides.

**Reply by 27shutterclicks** [reply]
Great, thank you very much.
